README.md

```diff
@@ -86,7 +86,7 @@ Once you have all dependencies in place, build VAST with the following commands:
 ```sh
 cmake -B build
 cmake --build build
-cmake --build build --target test
```

Comment:
Not sure how you can live with the broken symmetry for this one :D

Reply:
I'm not sure how I'll manage, might need to get some sort of emotional support. 🤕